projects/storefrontlib/shared/components/chat-messaging/messaging/messaging.component.ts

```diff
@@ -185,6 +203,18 @@ export class MessagingComponent implements OnInit, AfterViewChecked {
     }
   }
 
+  getMessageText(message: MessageEvent) {
```

Comment:
Please, add a comment to this method

Reply:
@Uli-Tiger, please, define a return data type

Reply:
@Uli-Tiger, please, mark resolved issues accordingly!

Reply:
its marked as resolved now....

Reply:
@Uli-Tiger, because I did it

Reply:
@Larisa-Staroverova: Well done!